docs/source/modules.rst

```diff
@@ -1,7 +1,6 @@
 .. initially generated by
    sphinx-apidoc -f -o source ../ ../versioneer.py ../setup.py
    (everything after the first "../" are exclude patterns)
-
```

Comment:
this causes
`/home/docs/checkouts/readthedocs.org/user_builds/pyfstat/checkouts/435/docs/source/modules.rst:4: WARNING: Explicit markup ends without a blank line; unexpected unindent`
so please reinstate the blank (can't do via suggestion, apparently)

Reply:
Done :+1: